compiler/rustc_parse/src/parser/stmt.rs

```diff
@@ -213,13 +213,21 @@ impl<'a> Parser<'a> {
         }
     }
 
-    fn recover_stmt_local(
+    fn recover_local_after_let(
         &mut self,
         lo: Span,
         attrs: AttrWrapper,
         subdiagnostic: fn(Span) -> InvalidVariableDeclarationSub,
     ) -> PResult<'a, Stmt> {
-        let stmt = self.recover_local_after_let(lo, attrs)?;
+        let stmt =
```

Comment:
I don't think it is reasonable to split it into a separate function (`recover_local_after_let`), so unified these functions here. In case this is not appropriate, please let me know.